lib/model/folder.go

```diff
@@ -450,6 +451,49 @@ func (f *folder) scanSubdirs(subDirs []string) error {
 
 		batch.append(res.File)
 		changes++
+
+		if res.Previous != nil && !res.Previous.IsDeleted() {
+			// Not a "new" file
```

Comment:
Why not check for deleted identical files for "non-new" files? I might just do `mv foo bar` where something already existed at `bar`.

Reply:
I was thinking about it, and there is no good reason to have this check, but I was concerned that there is no way to distinguish between `mv foo bar` and `touch bar`, and I am not sure if having a extra db lookup and potentially a few stats would be of concern.

I guess there is no good way to catch cases like:
```
mv b a
mv c b
```

as check of B for A would return the fact that B exists, but B is a completely different file C,
and given leveldb does not support dirty reads, I don't think there is a way to fix this.